content/en/blog/2025/announcing-etcd-3.6.md

```diff
@@ -0,0 +1,389 @@
+truth for membership data. In etcd v3.6.0, v2store can no longer be enabled as the `--enable-v2` flag has been removed,
+and v3store has become the sole source of truth for membership data.
+
+While v2store still exists in v3.6.0, etcd will fail to start if it contains any data other than membership information.
+To assist with migration, etcd v3.5.18+ provides the `etcdutl check v2store` command to verify if v2store contains any
+non-membership data (see [PR 19113][]).
+
+The removal of v2store is still ongoing and is tracked in [issues/12913][].
+
+### Downgrade
+etcd v3.6.0 is the first version to fully support downgrade. The effort for this downgrade task spans
+both versions 3.5 and 3.6, and all related work is tracked in [issues/11716][].
+
+At a high level, the process involves migrating the data schema to the target version (e.g., v3.5)
+followed by a rolling downgrade.
+
+Ensure the cluster is healthy, and take a snapshot backup. Validate whether the downgrade is valid:
+```
+$ etcdctl downgrade validate 3.5
+```
+
+If the downgrade is valid, enable downgrade mode:
+```
+$ etcdctl downgrade anable 3.5
+```
+
+etcd will then migrate the data schema in the background. Once complete, proceed with the rolling downgrade.
+
+For details, refer to the [Downgrade-3.6] guide.
+
+### Feature Gates
+Previously, experimental features used the `--experimental` prefix in their flag names. When stabilized, the
+prefix was removed, causing a breaking change. In etcd v3.6.0, feature gates were introduced, following Kubernetes'
+approach to managing new features. Features typically start in Alpha, progress to Beta, then GA, or get deprecated.
+See [feature-gates][] for details.
+
+### Livez/readyz checks
+etcd now supports `/livez` and `/readyz` endpoints, aligning with Kubernetes' Liveness and Readiness probes.
+`/livez` indicates whether the etcd instance is alive, while `/readyz` indicates when it is ready to serve
+requests. This feature has also been backported to release-3.5 since v3.5.11. See [livez/readyz][] for details.
+
+The existing `/health` endpoint remains functional. `/livez` is similar to `/health?serializable=true`, while
+`/readyz` is similar to `/health` or `/health?serializable=false`. Clearly, the `/livez` and `/readyz`
+endpoints provide clearer semantics and are easier to understand.
+
+### v3discovery
+In etcd v3.6.0, the new discovery protocol [v3discovery][] was introduced, based on clientv3.
+It facilitates the discovery of all cluster members during the bootstrap phase.
+
+The previous [v2discovery][] protocol, based on clientv2, has been deprecated. Additionally,
+the public discovery service at https://discovery.etcd.io/, which relied on v2discovery, is no longer maintained.
+
+## Breaking changes
+
+This section highlights a few notable breaking changes in this section. For a complete list, please refer to
+the [Upgrade etcd from 3.5 to 3.6][] and the [CHANGELOG-3.6][].
```

Comment:
Nit, since the title of the doc is "Upgrade etcd from v3.5 to v3.6", use the same one instead 3.5 to 3.6 ( I guess the old title didn't use version with v so that's why you omitted them).